demo/src/source-info.js

```diff
@@ -175,8 +177,9 @@ const remoteBFTiff = {
   description: 'Tonsil Legacy Bioformats Pyramid Tiff'
 };
 
+// Generated using bioformats2raw and raw2ometiff.
 const remoteTiffUrl2 =
-  'https://vitessce-demo-data.storage.googleapis.com/test-data/VAN0003-LK-32-21-AF_preIMS_registered.pyramid.ome.tiff ';
+  'https://vitessce-demo-data.storage.googleapis.com/test-data/test_VAN0003-LK-32-21-AF_preIMS_registered.ome.tif';
```

Comment:
Sorry just now to be noticing this, but we don't seem to have a versioning directory structure under `vitessce-demo-data.storage.googleapis.com`... I think it would be a good thing here, for the same reasons it's good on S3.

Reply:
I don't disagree, but this data is actually internal/not-public for now.  I'll make an issue for it.